reviewdog.go

```diff
@@ -120,6 +120,10 @@ func (w *Reviewdog) Run(r io.Reader) error {
 	return nil
 }
 
+var unused = ""
+
+var NoComment = ""
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported var NoComment should have comment or be unexported